src/pages/ContentScripts/features/developer-avator-openrank/index.tsx

```diff
@@ -0,0 +1,66 @@
+import features from '../../../../feature-manager';
+import { getOpenrank } from '../../../../api/developer';
+import elementReady from 'element-ready';
+
+const featureId = features.getFeatureID(import.meta.url);
+
+const getData = async (developerName: string): Promise<string | null> => {
+  const data = await getOpenrank(developerName);
+  if (data) {
+    const keys = Object.keys(data);
+    const latestKey = keys[keys.length - 1];
+    return data[latestKey];
+  }
+  return null;
+};
```

Comment:
Change the function name to `getDeveloperLatestOpenrank` or just `getLatestOpenrank`. `getData` is too general.

Also, can we use Object.values to get the lastest openrank? Why are we getting the latest key first?

Reply:
already edited and i use Object.values instead of Object.key now